native/packages/react-native-bpk-component-section-list/stories.js

```diff
@@ -78,48 +78,73 @@ const getFlagUriFromCountryCode = countryCode =>
 
 // eslint-disable-next-line react/no-multi-comp
 class StatefulBpkSectionList extends React.Component<{
+  extraEntries: number,
   showImages: boolean,
 }> {
   static propTypes = {
+    extraEntries: PropTypes.number,
     showImages: PropTypes.bool,
   };
 
   static defaultProps = {
+    extraEntries: 0,
     showImages: false,
   };
 
   constructor() {
     super();
     this.state = { selectedAirport: 'GLA' };
   }
+
+  onItemPress = item => {
+    this.setState({ selectedAirport: item.id });
+  };
+
+  getData = () => {
+    const data = airportCities.slice();
+    if (this.props.extraEntries > 0) {
+      const newAirports = [];
+      for (let i = 0; i < this.props.extraEntries; i += 1) {
```

Comment:
You can use `new Array(this.props.extraEntries).fill().map(...)` here...

Reply:
Oh yeah, cheers!